IntellitectTerminal.Web/src/views/Home.vue

```diff
@@ -2,466 +2,63 @@
-        }
-
-        // Directory level navigation. Endpoint must be inside of this.path
-        // Get each direction we need from the arg[0]. EX ./user/file [user,file]
-        let directions = arg[0].split("/").filter(element => element == "" || element == "." ? false : true);
-
-        this.updatePath(validatePath(this.pathTree, directions, this.term, arg[0]));
-        break;
-
-      // TODO: cat into a direct path ex: cat /home/user/readme.txt
-      case Commands.CAT:
-        // Arg[0] is required
-        if (arg[0] == undefined) {
-          output(Commands.CAT, this.term, "Missing file argument path");
-          break;
-        }
-
-        // Find the file
-        let file: TreeNode | null = null;
-        this.pathTree.Children.forEach((child) => file = (child.Name == arg[0]) ? child : null);
-        if (file == null) {
-          output(Commands.CAT, this.term, `File not found '${arg[0]}'`)
-          break;
-        }
-        if (!(file as TreeNode).isFile) {
-          output(Commands.CAT, this.term, "Argument is a directory and not a file");
-        }
-        await this.commandservice.cat(this.user?.userId!, (file as TreeNode).Name);
-        let catouput = this.commandservice.cat.result;
-        this.term.writeln("");
-        this.term.writeln(`${catouput}`);
-        break;
-
-      case Commands.CLEAR:
-        if (arg.length > 0) {
-          unknownArg(Commands.HELP, this.term, arg[0]);
-          break;
-        }
-        this.term.clear();
-        break;
-
-      case Commands.PROGRESS:
-        if (arg.length > 0) {
-          unknownArg(Commands.HELP, this.term, arg[0]);
-          break;
-        }
-        await this.commandservice.progress(this.user?.userId!);
-        let progressoutput = this.commandservice.progress.result;
-        this.term.writeln(`${progressoutput}`);
-        break;
-
-      case Commands.PWD:
-        if (arg.length > 0) {
-          unknownArg(Commands.HELP, this.term, arg[0]);
-          break;
-        }
-        this.term.writeln(this.pwd);
-        break;
-
-      case Commands.SUBMIT:
-
-        if (arg.length > 0) {
-          unknownArg(Commands.HELP, this.term, arg[0]);
-          break;
-        }
-
-        // Click the input element
-        this.doc?.click();
-        break;
+      if (user != null) {
 
-      case Commands.VERIFY:
-        if (arg.length > 0) {
-          unknownArg(Commands.HELP, this.term, arg[0]);
-          break;
-        }
-        await this.commandservice.verify(this.user!.userId!);
-        if (this.commandservice.verify.result) {
-          output("intelliterm", this.term, "Successful output! Run request to get another challenge");
-          this.term.write(this.hostname);
-          break;
-        }
-        output("intelliterm", this.term, "Incorrect output. Submit another file ");
-        this.term.write(this.hostname);
-        break;
+        if(userId != null) {
+          setCookie('userId', userId, { expires: 100 });
```

Comment:
```suggestion
          Cookies.setCookie('userId', userId, { expires: 100 });
```
I missed one other place